ts/src/hyperliquid.ts

```diff
@@ -931,7 +931,10 @@ export default class hyperliquid extends Exchange {
             'signature': signature,
             // 'vaultAddress': vaultAddress,
         };
-        const response = await this.privatePostExchange (this.extend (request, params));
+        if (vaultAddress !== undefined) {
```

Comment:
two things, we're missing the part where we extend params with the orderobject, and a test using vaultAddress